setup/includes/upgrades/common/3.0.0-update-legacy-class-references.php

```diff
@@ -78,3 +80,9 @@
 $modx->updateCollection($class, ['class_key' => modFileMediaSource::class], ['class_key' => 'sources.modFileMediaSource']);
 $modx->updateCollection($class, ['class_key' => modFTPMediaSource::class], ['class_key' => 'sources.modFTPMediaSource']);
 $modx->updateCollection($class, ['class_key' => modS3MediaSource::class], ['class_key' => 'sources.modS3MediaSource']);
+
+/* modify core class references in modContextSettings */
+$class = modContextSetting::class;
+$table = $modx->getTableName($class);
+
+$modx->updateCollection($class, ['value' => modManagerRequest::class], ['key' => 'modRequest.class']);
```

Comment:
Should this, like the other examples above, only update where the _value_ is `modManagerRequest` rather than always change the `modRequest.class` setting?

Reply:
Yes, you are right!